pynest/nest/server/hl_api_server.py

```diff
@@ -249,17 +253,25 @@ def get_arguments(request):
     return list(args), kwargs
 
 
-def get_globals():
-    """ Get globals for exec function.
-    """
-    copied_globals = globals().copy()
-
-    # Add modules to copied globals
-    modlist = [(module, importlib.import_module(module)) for module in MODULES]
-    modules = dict(modlist)
-    copied_globals.update(modules)
+def get_modules_from_env():
+    """ Get modules from env NEST_SERVER_MODULES.
 
-    return copied_globals
+        It converts environment variable NEST_SERVER_MODULES:
+            `NEST_SERVER_MODULES="nest,numpy as np,random from numpy"`
+        to formatted dictionary for updating globals:
+            `{'nest': <module 'nest'> 'np': <module 'numpy'>, 'random': <module 'numpy.random'>}`
+    """
+    modules = {}
+    for module in MODULES:
+        if ' as ' in module:
+            modname, modvar = module.split(' as ')
+            modules[modvar] = importlib.import_module(modname)
+        elif ' from ' in module:
+            modvar, modname = module.split(' from ')
```

Comment:
We could even support multiple fields to be imported by allowing `modvar` to be a comma separated list and iterating it here.

But maybe that's too much. You decide. I just wanted to raise my idea... :wink:

Reply:
Thank you for dropping your idea. Can you please give some examples for your proposal?


Inspired from Python imports such as:
```
import nest
import numpy as np
from numpy import random
```
the environment variables could be follow similar patterns:
```
export NEST_SERVER_MODULES="nest,numpy as np,random from numpy"
```